tests/kafkatest/services/templates/kafka.properties

```diff
@@ -20,6 +20,8 @@ port=9092
 #host.name=localhost
 advertised.host.name={{ node.account.hostname }}
 #advertised.port=<port accessible by clients>
+listeners=PLAINTEXT://:9092,SSL://:9093
```

Comment:
In the current patch, end-to-end latency tests are run with (protocol=PLAINTEXT,interbroker_protocol=PLAINTEXT), (protocol=SSL,interbroker_protocol=PLAINTEXT) and (protocol=SSL,interbroker_protocol=SSL). All other performance tests are run with (PLAINTEXT, PLAINTEXT and (SSL,SSL). I will add (SSL, PLAINTEXT) as well to all of the performance tests since it is quite easy to do.